sympy/core/tests/test_sympify.py

```diff
@@ -722,22 +722,22 @@ def test_issue_14706():
     if not numpy:
         skip("numpy not installed.")
 
-    z1 = numpy.zeros((1, 1), dtype=numpy.float)
-    z2 = numpy.zeros((2, 2), dtype=numpy.float)
-    z3 = numpy.zeros((), dtype=numpy.float)
+    z1 = numpy.zeros((1, 1), dtype=numpy.float64)
+    z2 = numpy.zeros((2, 2), dtype=numpy.float64)
+    z3 = numpy.zeros((), dtype=numpy.float64)
 
-    y1 = numpy.ones((1, 1), dtype=numpy.float)
-    y2 = numpy.ones((2, 2), dtype=numpy.float)
-    y3 = numpy.ones((), dtype=numpy.float)
+    y1 = numpy.ones((1, 1), dtype=numpy.float64)
+    y2 = numpy.ones((2, 2), dtype=numpy.float64)
+    y3 = numpy.ones((), dtype=numpy.float64)
 
     assert numpy.all(x + z1 == numpy.full((1, 1), x))
     assert numpy.all(x + z2 == numpy.full((2, 2), x))
     assert numpy.all(z1 + x == numpy.full((1, 1), x))
     assert numpy.all(z2 + x == numpy.full((2, 2), x))
     for z in [z3,
-              numpy.int(0),
-              numpy.float(0),
-              numpy.complex(0)]:
+              int(0),
+              numpy.float64(0),
+              complex(0)]:
```

Comment:
This test is supposed to test for numpy types so it should be e.g. `numpy.int64` rather than `int`.

Reply:
Hi, I have pushed the necessary changes but my PR can't seem to get past the code-quality check (I get the message 'make sure all commits have an associated author'). I didn't have this error previously, could you please advise a solution?